action.yml

```diff
@@ -0,0 +1,21 @@
+name: 'Azure SDK Action'
```

Comment:
Can we go ahead and setup the structure such that this not in the root but instead a sub-folder so others can move new actions here without needing to update this.

Reply:
Let me try this out.

Reply:
From what I can tell, external repo actions don't support sub-paths, just an optional git ref ([docs](https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#runsstepsuses)). However we could support `with` parameters to the top level action to specify a sub-action, and then we can call `uses` from the main action to reference other `action.yml` files in the repository.